pandas/tests/io/excel/test_writers.py

```diff
@@ -1223,7 +1222,7 @@ class TestExcelWriterEngineTests:
             pytest.param(_XlwtWriter, ".xls", marks=td.skip_if_no("xlwt")),
         ],
     )
-    def test_ExcelWriter_dispatch(self, klass, ext):
+    def test_ExcelWriter_dispatch(self, klass, ext, path):
         with ensure_clean(ext) as path:
             writer = ExcelWriter(path)
```

Comment:
This confuses me here.  Why are we passing in the `path` fixture if only to overwrite with this `with` block?

Reply:
Refactored this, was an error